src/main/scala/org/broadinstitute/dsde/vault/services/uBAMCollection/UBamCollectionIngestService.scala

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.dsde.vault.services.uBAMCollection
+
+import com.wordnik.swagger.annotations._
+import org.broadinstitute.dsde.vault.DmClientService
+import org.broadinstitute.dsde.vault.common.directives.VersioningDirectives._
+import spray.httpx.SprayJsonSupport._
+import org.broadinstitute.dsde.vault.model.uBAMCollectionJsonProtocol._
+import org.broadinstitute.dsde.vault.model.{UBamCollectionIngest, UBamCollectionIngestResponse}
+import org.broadinstitute.dsde.vault.services.VaultDirectives
+import spray.routing.HttpService
+
+@Api(value = "/collections", description = "Collections Service", produces = "application/json")
```

Comment:
A little hard to review with changes still floating out in DM pull requests, but this doesn't match the other endpoint names. This test can't pass as is, even against the DM 28 branch. (Assuming that access to BOSS via VPN is a blocker for running `sbt test`, will be nice to get mock up and running from #48.)